pkgs/clean-pkg/src/genie/libs/clean/stages/iosxe/stages.py

```diff
@@ -317,7 +317,7 @@ def tftp_boot(section, steps, device, ip_address, subnet_mask, gateway,
         # to the device. Cannot use device.reload() directly as in case of HA,
         # we need both sup to do the commands
         device.sendline('reload')
-        device.sendline('yes')
```

Comment:
Hi Karim,
 
We could provide flag but not sure if it’s really needed.  There is no flag for the reload clean stage.  I would think tftp_boot should behave similarly with reload.
 
Would you have pointers how to test the change? 
 
Thanks,
Tom

Reply:
Hi Tom,

As discussed over Webex, to provide a full run with your change you need to be in developer mode under clean package.
 
Steps to be in developer mode;
 

- pip uninstall -y genie.libs.clean

- git clone https://github.com/CiscoTestAutomation/genielibs.git   (Under pypi folder if not there, create one, not mnadatory but that’s how we recommend it)

- cd <pyats env>/pypi/genielibs/pkgs/clean-pkg

- python setup.py develop  ----- > now you are in developer mode

- Make your changes in that location (<pyats env>/pypi/genielibs/pkgs/clean-pkg) and every run you trigger from your env will run with your changes

 
Since the change will affect current users we recommend to have it backward compatible by adding an argument (in tftp_boot stage) where user will have the choice to device.sendline(‘no’), default will be device.sendline('yes').
 
If you face any issue, we can implement that argument for you and test the change but we will need the testbed yaml file along with the clean file yaml used.
 
Thanks,
Karim Mohamed